src/create-render-context.js

```diff
@@ -172,6 +176,21 @@ const patchTimers = (): void => {
     patchCallbackFnWithGate(window, "requestAnimationFrame", "__SSR_ACTIVE__");
 };
 
+const createVirtualConsole = () => {
+    const virtualConsole = new VirtualConsole();
+    virtualConsole.on("jsdomError", (e: Error) => {
+        if (e.message.indexOf("Could not load img") >= 0) {
+            // We know that images cannot load. We're deliberately blocking
+            // them.
+            return;
+        }
```

Comment:
Are there other file types that we might want to block as well like font files?

Reply:
We are already blocking them, we're just not silencing the error, because I haven't yet seen an error that needs silencing.